server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/Bounce.java

```diff
@@ -51,7 +51,7 @@
  * A notice text can be specified, and in such case will be inserted into the
  * notification inline text.<br>
  * If the notified message has an "error message" set, it will be inserted into
- * the notification inline text. If the <code>attachStackTrace</code> init
+ * the notification inline text. If the <code>attachError</code> init
```

Comment:
If you modify init parameters name, then it might have some effect on the website